guides/data/git.md

```diff
@@ -1,8 +1,134 @@
-[git](https://git-scm.com/) is the default data source for Cards.
+With the [git plugin](https://github.com/cardstack/cardstack/tree/master/packages/git) for the Card SDK, you can store your card's data as JSON in a local or remote git repository.
+You get an instant, versioned data persistence layer that supports create, read, update, and delete.
 
-git has many advantages, including versioning, the ability to compare current and past states, and powerful rollback capabilities.
+In this guide, we will show how to save Card data to git by default.
+
+## Installing and configuring
+
+Install [`@cardstack/git`](https://github.com/cardstack/cardstack/tree/master/packages/git) in the `cardhost` of your project:
+
+```sh
+cd cardhost
+yarn add --dev @cardstack/git
```

Comment:
shouldn't this be `yarn install --dev @cardstack/git`? When I did `yarn add` it complained about `@cardstack/git` not being installed when I was trying to run `ember generate card foo` to generate a card.